tests.py

```diff
@@ -86,6 +86,12 @@ def test_create_envelope_from_document_request(self):
             '\r\n'
         ))
 
+    def test_timeout(self):
```

Comment:
Add docstring where first line summarizes what you expect (behaviour driven style).
Something like `"""DocuSignClient with (too small) timeout raises exception when issuing requests."""`